app/controllers/other/add/DiscretionYesNoController.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.other.add
+
+import config.annotations.AddOtherBeneficiary
+import connectors.TrustConnector
+import controllers.actions.StandardActionSets
+import controllers.actions.other.DescriptionRequiredAction
+import forms.YesNoFormProvider
+import javax.inject.Inject
+import navigation.Navigator
+import pages.other.DiscretionYesNoPage
+import play.api.data.Form
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.PlaybackRepository
+import services.TrustService
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import views.html.other.add.DiscretionYesNoView
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DiscretionYesNoController @Inject()(
+                                           val controllerComponents: MessagesControllerComponents,
+                                           standardActionSets: StandardActionSets,
+                                           formProvider: YesNoFormProvider,
+                                           connector: TrustConnector,
+                                           view: DiscretionYesNoView,
+                                           trustService: TrustService,
+                                           repository: PlaybackRepository,
+                                           @AddOtherBeneficiary navigator: Navigator,
+                                           nameAction: DescriptionRequiredAction
```

Comment:
Not a huge deal but could this be descriptionAction? Applies to all controllers I suspect

Reply:
Ah yeah, i did intend to change that but forgot